src/aind_behavior_experiment_launcher/behavior_launcher/slims_picker.py

```diff
@@ -0,0 +1,270 @@
+        Prompts the user to provide a rig name.
+
+        Returns:
+            _R: The selected rig configuration.
+
+        Raises:
+            SlimsRecordNotFound: If no rig is found in Slims or an invalid choice is made.
+        """
+
+        while True:
+            instruments = self.slims_client.fetch_models(models.SlimsInstrument)[-100:]
+            rig = None
+            while rig is None:
+                rig = self.ui_helper.input("Enter rig name: ")
+                if rig == "":
+                    rig = self.ui_helper.prompt_pick_from_list(
+                        [instrument.name for instrument in instruments],
+                        prompt="Choose a rig:",
+                        allow_0_as_none=True,
+                    )
+                try:
+                    slims_rig = self.slims_client.fetch_model(models.SlimsInstrument, name=rig)
+                    return self.launcher.rig_schema_model(rig_name=slims_rig.name)
+
+                except exceptions.SlimsRecordNotFound as e:
+                    logger.error(f"Rig {rig} not found in Slims. Try again.")
+                    rig = None
+
+    def pick_session(self) -> _S:
+        """
+        Prompts the user to select or create a session configuration.
+
+        Returns:
+            TSession: The created or selected session configuration.
+
+        Raises:
+            ValueError: If no session model is found on slims.
+        """
+
+        username = self.prompt_username(strict=True)
+        if self.launcher.subject is not None:
+            logging.info("Subject provided via CLABE: %s", self.launcher.settings.subject)
+            subject = self.launcher.subject
+        else:
+            slims_mice = self.slims_client.fetch_models(models.SlimsMouseContent)[
+                -100:
+            ]  # grab 100 latest mice from slims
+            subject = None
+            while subject is None:
+                subject = self.ui_helper.input("Enter subject name: ")
+                if subject == "":
+                    subject = self.ui_helper.prompt_pick_from_list(
+                        [mouse.barcode for mouse in slims_mice],
+                        prompt="Choose a subject:",
+                        allow_0_as_none=True,
+                    )
+                try:
+                    self._slims_mouse = self.slims_client.fetch_model(models.SlimsMouseContent, barcode=subject)
+                except exceptions.SlimsRecordNotFound:
+                    logger.info(f"No Slims mouse with barcode {subject}. Please re-enter.")
+                    subject = None
+            self.launcher.subject = subject
+
+        try:
+            self._slims_session = self.slims_client.fetch_models(
+                models.behavior_session.SlimsBehaviorSession, mouse_pk=self._slims_mouse.pk
+            )[-1]
+        except IndexError:  # empty list returned from slims
+            raise ValueError(f"No session found on slims for mouse {subject}.")
+
```

Comment:
Add 
`        notes = self.ui_helper.prompt_text("Enter notes: ")
`
Append these notes to slims notes before writing the session model